sig-policy/policy-charter.md

```diff
@@ -0,0 +1,35 @@
+# SIG Policy charter
+
+## Scope
+
+SIG Policy covers governance capability, security remediation, and configuration management for the Open Cluster Management (OCM) project.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Review and approve policy related OCM enhancements.
+- APIs used for OCM policies.
+- Tools and documentation to aid in ecosystem tool interoperability around policy (e.g. Policy Generator)
```

Comment:
let's say someone wants to do an integration of OPA with ocm, will it fall into policy group's scope?

Reply:
Yes. This is now covered in the Scope section.